src/Orleans/CodeGeneration/ConsiderForCodeGenerationAttribute.cs

```diff
@@ -3,18 +3,25 @@ namespace Orleans.CodeGeneration
     using System;
     
     /// <summary>
-    /// Abstract base class for attributes that effect code generation of types.
+    /// The attribute which informs the code generator that code should be generated for this type.
     /// </summary>
-    public abstract class ConsiderForCodeGenerationAttribute : Attribute
```

Comment:
Was originally marked abstract to communicate to maintainers that this class was intended as a base class, and not to be used on it's own.  No fields were actually abstract.  I don't have strong opinion about it not being abstract any longer, but I request you consider how to help prevent this attribute, which is public, from being misunderstood.